services/app/src/components/team/Team.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import TeamMember from './TeamMember';
+import playerStatuses from '../../constants/playerStatuses';
+
+function Team({ title, users, skipMove }) {
+  return (
+    <div style={{ marginBottom: '10px' }}>
+      <h2>{title}</h2>
+      <div>
+        {users.map((user) => (
+          <TeamMember
+            key={user.id}
+            name={user.name}
+            id={user.id}
+            skipMove={skipMove}
+            status={user.status}
+          />
+        ))}
+      </div>
+    </div>
+  );
+}
+
+Team.propTypes = {
+  title: PropTypes.string.isRequired,
+  users: PropTypes.arrayOf(
+    PropTypes.shape({
```

Comment:
Can we extract `user` PropType to a separate constant and reuse it across the app?